apisix/schema_def.lua

```diff
@@ -956,6 +956,11 @@ _M.plugin_injected_schema = {
                 description = "priority of plugins by customized order",
                 type = "integer",
             },
+            filter = {
+                description = "filter determines whether the plugin "..
+                                "needs to be executed at runtime",
+                type  = "array",
+            }
```

Comment:
```suggestion
            filter = {
                description = "filter determines whether the plugin "..
                                "needs to be executed at runtime",
                type  = "array",
                maxItems = 20,
                items = {
                    type = "array",
                },
            }
```

Reply:
What is the basis for this (`maxItems = 20`) restriction?

Reply:
It is just a convention within APISIX to create expr, which in general does not require many child elements.

Reply:
I think this can be omitted first, it has no effect on apisix, and users can choose by themselves. What do you think?

Reply:
From a prudential point of view, I would like to add, in order to gradually unify.